app/scripts/migrations/048.js

```diff
@@ -20,14 +22,30 @@ export default {
   },
 }
 
+const hexRegEx = (/^0x[0-9a-f]+$/ui)
+const chainIdRegEx = (/^0x[1-9a-f]+[0-9a-f]*$/ui)
```

Comment:
What about capitals? `parseInt` seems to be tolerant of them

```suggestion
const hexRegEx = (/^0x[0-9a-fA-F]+$/ui)
const chainIdRegEx = (/^0x[1-9a-fA-F]+[0-9a-fA-F]*$/ui)
```

Reply:
Hmm, we don't allow capitals when checking this in the UI either. So this suggestion is no good - it'd be a mismatch. And it makes sense to disallow them for the purpose of normalization.

Reply:
New suggestion:
```suggestion
const hexRegEx = (/^0x[0-9a-fA-F]+$/ui)
const chainIdRegEx = (/^0x[1-9a-f]+[0-9a-f]*$/ui)
```

The `chainIdRegEx` can stay as-is because it strictly expects the `chainId` to be formatted exactly as we save it internally, but the `hexRegEx` doesn't need to be that strict I don't think.

Reply:
Oh, I see the `i` flag now. Nevermind!